pkg/transforms/transformer.go

```diff
@@ -356,6 +355,19 @@ func TransformRoutine(input chan *Event, output chan NodeEvent) {
 			}
 			trans = PolicyResourceBuilder(&typedResource)
 
+		case [2]string{"ConfigurationPolicy", "policy.open-cluster-management.io"}:
+			trans = ConfigPolicyResourceBuilder(event.Resource)
+
+		case [2]string{"OperatorPolicy", "policy.open-cluster-management.io"}:
+			var err error
+			trans, err = OperatorPolicyResourceBuilder(event.Resource)
+			if err != nil {
+				panic(err) // Will be caught by handleRoutineExit
+			}
```

Comment:
Why is there an error being checked here? I don't see anything other than `nil` being returned in the function.